frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -0,0 +1,130 @@
+const statusColumnClasses = 'col-lg-2 col-md-4 hidden-sm hidden-xs';
+const machineColumnClasses = 'col-lg-3 visible-lg';
+const roleColumnClasses = 'col-lg-2 visible-lg';
+const addressColumnClasses = 'col-lg-2 visible-lg';
+
+const HostHeader: React.FC<React.ComponentProps<typeof ColHead>> = props => (
+  <ListHeader>
+    <ColHead {...props} className={nameColumnClasses} sortField="metadata.name">
+      Name
+    </ColHead>
+    <ColHead {...props} className={statusColumnClasses}>
+      Status
+    </ColHead>
+    <ColHead {...props} className={machineColumnClasses}>
+      Machine
+    </ColHead>
+    <ColHead {...props} className={roleColumnClasses}>
+      Role
+    </ColHead>
+    <ColHead {...props} className={addressColumnClasses} sortField="spec.bmc.address">
+      Management Address
+    </ColHead>
+  </ListHeader>
+);
+
+const HostRow: React.FC<React.ComponentProps<typeof ResourceRow>> = ({ obj: host }) => {
+  const name = getName(host);
+  const namespace = getNamespace(host);
+  const uid = getUid(host);
+  // const machineName = getHostMachineName(host);
+  const address = getHostBmcAddress(host);
+
+  // TODO(jtomasek): other resource references will be updated as a subsequent change
+  // const machineResource = {
+  //   kind: referenceForModel(MachineModel),
+  //   name: machineName,
+  //   namespaced: true,
+  //   namespace,
+  //   isList: false,
+  //   prop: 'machine',
+  // };
+
+  // const hostResourceMap = {
+  //   machine: {
+  //     resource: machineResource,
+  //   },
+  //   nodes: {
+  //     resource: getResource(NodeModel, { namespaced: false }),
+  //   },
+  // };
+
+  return (
+    <ResourceRow obj={host}>
+      <div className={nameColumnClasses}>
+        <ResourceLink kind={BaremetalHostModel.kind} name={name} namespace={namespace} title={uid} />
+      </div>
+      <div className={statusColumnClasses}>
+        {/* <WithResources resourceMap={machineName ? hostResourceMap : {}}>
+          <BaremetalHostStatus host={host} />
+        </WithResources> */}
+      </div>
+      <div className={machineColumnClasses}>
+        <MachineCell host={host} />
+      </div>
+      <div className={roleColumnClasses}>
+        {/* <WithResources resourceMap={machineName ? hostResourceMap : {}}>
+          <BaremetalHostRole />
+        </WithResources> */}
+      </div>
+      <div className={addressColumnClasses}>{address}</div>
+    </ResourceRow>
+  );
+};
+
+const HostList: React.FC<React.ComponentProps<typeof List>> = props => (
+  <List {...props} Header={HostHeader} Row={HostRow} />
+);
+
+// TODO(jtomasek): re-enable filters once the extension point for list.tsx is in place
+const filters = [];
+// const filters = [
+//   {
+//     type: 'baremetalhost-status',
+//     selected: ['online', 'offline'],
+//     reducer: getSimpleHostStatus,
+//     items: [{ id: 'online', title: 'online' }, { id: 'offline', title: 'offline' }],
+//   },
+// ];
+
+/* eslint-disable no-unused-vars, no-undef */
```

Comment:
You should be able to remove this comment.

Reply:
Done